service/controller/resource/servicemonitor/service/apiserver.go

```diff
@@ -37,14 +37,6 @@ func APIServer(cluster metav1.Object, provider string, installation string) *pro
 							SourceLabels: []string{"__meta_kubernetes_service_name"},
 							TargetLabel:  "app",
 						},
-						{
-							SourceLabels: []string{"__meta_kubernetes_namespace"},
-							TargetLabel:  "namespace",
-						},
-						{
-							SourceLabels: []string{"__meta_kubernetes_pod_name"},
-							TargetLabel:  "pod_name",
-						},
```

Comment:
Why are those gone ?

Reply:
Prometheus operator is adding them for ServiceMonitors

Reply:
https://github.com/prometheus-operator/prometheus-operator/blob/55baf034c431ed2c78d950b187f7d8b34dd06860/pkg/prometheus/promcfg.go#L567